resources/devices/mobile-phone/htc.json

```diff
@@ -1057,7 +1057,7 @@
         "properties": {
             "Device_Type": "Mobile Phone",
             "Device_Code_Name": "M7",
-            "Device_Name": "One",
+            "Device_Name": "One M7",
```

Comment:
Technically the M7 is just "HTC One", but I thought it was helpful to rename this to prevent any confusion.